discovery-provider/src/tasks/entity_manager/entities/user.py

```diff
@@ -179,11 +196,9 @@ def create_user(params: ManageEntityParameters, cid_type: Dict[str, str], cid_me
             params.web3,
             params.challenge_bus,
         )
-        metadata_type, _ = get_metadata_type_and_format(
-            params.entity_type
-        )
+        metadata_type, _ = get_metadata_type_and_format(params.entity_type)
         cid_type[metadata_cid] = metadata_type
-        cid_metadata[metadata_cid] = params.metadata
+        cid_metadata[metadata_cid] = user_metadata
```

Comment:
We were storing the unparsed metadata, which is a string. Not sure if this has been broken or if I broke it somehow. Now storing the parsed metadata, which works properly

Reply:
nice catch - `params.metadata` here isnt actually parsed yet as opposed to when we do `cid_metadata[metadata_cid] = params.metadata` in entity_manager.py